packages/meteor/debug.js

```diff
@@ -1,67 +1,76 @@
-var suppress = 0;
+let suppress = 0;
 
-// replacement for console.log. This is a temporary API. We should
-// provide a real logging API soon (possibly just a polyfill for
-// console?)
-//
-// NOTE: this is used on the server to print the warning about
-// having autopublish enabled when you probably meant to turn it
-// off. it's not really the proper use of something called
-// _debug. the intent is for this message to go to the terminal and
-// be very visible. if you change _debug to go someplace else, etc,
-// please fix the autopublish code to do something reasonable.
-//
-Meteor._debug = function (/* arguments */) {
+/**
+ * Replacement for console.log. This is a temporary API. We should
+ * provide a real logging API soon (possibly just a polyfill for
+ * console?)
+ *
+ * NOTE: this is used on the server to print the warning about
+ * having autopublish enabled when you probably meant to turn it
+ * off. it's not really the proper use of something called
+ * _debug. the intent is for this message to go to the terminal and
+ * be very visible. if you change _debug to go someplace else, etc,
+ * please fix the autopublish code to do something reasonable.
+ */
+Meteor._debug = function(...args) {
   if (suppress) {
     suppress--;
     return;
   }
-  if (typeof console !== 'undefined' &&
-      typeof console.log !== 'undefined') {
-    if (arguments.length == 0) { // IE Companion breaks otherwise
-      // IE10 PP4 requires at least one argument
-      console.log('');
-    } else {
-      // IE doesn't have console.log.apply, it's not a real Object.
-      // http://stackoverflow.com/questions/5538972/console-log-apply-not-working-in-ie9
-      // http://patik.com/blog/complete-cross-browser-console-log/
-      if (typeof console.log.apply === "function") {
-        // Most browsers
 
-        // Chrome and Safari only hyperlink URLs to source files in first argument of
-        // console.log, so try to call it with one argument if possible.
-        // Approach taken here: If all arguments are strings, join them on space.
-        // See https://github.com/meteor/meteor/pull/732#issuecomment-13975991
-        var allArgumentsOfTypeString = true;
-        for (var i = 0; i < arguments.length; i++)
-          if (typeof arguments[i] !== "string")
-            allArgumentsOfTypeString = false;
+  if (typeof console !== 'undefined' && typeof console.log !== 'undefined') {
+    return;
+  }
 
-        if (allArgumentsOfTypeString)
-          console.log.apply(console, [Array.prototype.join.call(arguments, " ")]);
-        else
-          console.log.apply(console, arguments);
+  if (args.length === 0) { // IE Companion breaks otherwise
+    // IE10 PP4 requires at least one argument
+    console.log('');
+  }
+  else {
```

Comment:
Please keep `else`'s and `else if`'s on the same line as the closing/opening brace. E.g. `} else {`

Reply:
oh please https://github.com/meteor/meteor/pull/9680

Reply:
I'm not planning fixing that since you should provide some `auto code style format tool` first

Reply:
@pravdomil We're not interested in adopting an auto-code style tool at this time.  The Meteor codebase started way before such tools were popularized and running one in bulk on our codebase would destroy too much valuable commit history.  Linters are great when a project starts with them and has always had them in place, but when they introduce additional walls between debugging when a broken commit was introduced, or mask history (by requiring to dive through "Linter" commits), they don't do much for an existing codebase.

If you're not interested in making the change, we'll do it, but I hope you understand.  For what it's worth, we do have a guide in place which makes the same request that @hwillson has made here: https://github.com/meteor/javascript#16.2.

Reply:
>  running one in bulk on our codebase would destroy too much valuable commit history

agree, but there is way just in between "no formatting" and "auto formatting", what I would propose here is, setup prettier or other auto formatting tools, so that *new code* will be written in same style and we don't need to worry about that anymore and spend time on more productive things

now you provide no code style and it's hard to write new code without getting comment like this one